packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -379,8 +379,9 @@ export class RunningSummarizer implements IDisposable {
                     this.logger.sendPerformanceEvent({
                         eventName: "SummarizeAttemptDelay",
                         duration: delaySeconds,
-                        reason: overrideDelaySeconds !== undefined ? "nack with retryAfter" : undefined,
                         ...summarizeProps,
+                        // summarizeProps has its own reason.
+                        reason: overrideDelaySeconds !== undefined ? "nack with retryAfter" : summarizeProps.reason,
```

Comment:
Before latest rename, we used to have reason & summarizeReason. They had slightly different meanings.
I'd say that we should not merge them into one property here.
How about we do something like

```ts
summaryNackDelay: overrideDelaySeconds !== undefined,
```
This seems like sufficient.